PC/python3.def

```diff
@@ -87,6 +87,9 @@ EXPORTS
   PyDictItems_Type=python38.PyDictItems_Type DATA
   PyDictIterItem_Type=python38.PyDictIterItem_Type DATA
   PyDictIterKey_Type=python38.PyDictIterKey_Type DATA
+  PyDictRevIterKey_Type=python38.PyDictRevIterKey_Type DATA
+  PyDictRevIterItem_Type=python38.PyDictRevIterItem_Type DATA
+  PyDictRevIterValue_Type=python38.PyDictRevIterValue_Type DATA
```

Comment:
Should these values be exposed by DLL?

Reply:
Hi, if my memory is good (so I may absolutly be wrong) tests were not passing on windows before I added the types in this file. PyListRevIter_Type is here too so I thought those three new types may need to be here too but I'm not sure what puporse this file serves and if it is really necessary.